README.md

```diff
@@ -15,8 +15,9 @@ That API-key should have CDF capabilities required to run the code within the Fu
 1. `cdf_project`: Name of your CDF project/tenant. Inferred from your API-keys. Will be validated with API-keys if provided
 2. `cdf_base_url`: Base url of your CDF tenant, defaults to _https://api.cognitedata.com_
 3. `function_file`: Name of the main function python file (defaults to `handler.py`)
-4. `schedule_file`: File location with the list of schedules to be applied. see file format below. (defaults to None ie no schedules)
-5. `remove_only`: Checks that specified function is removed with all it's schedules. Deployment logic is skipped
+4. `function_secrets`: Name of Github secrets that holds the base64 encoded json dictionary with secrets.
```

Comment:
Name of THE Github secret .....
So the json dictionary needs to be base64 encoded in the secret?

Reply:
yup. you can store whatever and we don't need to think much about what is in there